Platforms/QemuQ35Pkg/.azurepipelines/NuGet.yml

```diff
@@ -0,0 +1,204 @@
+## @file
+# Azure Pipeline build file for publishing a NuGet package.
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+parameters:
+- name: publish_q35_binaries
+  displayName: Publish QEMU Q35 Binaries
+  type: boolean
+  default: true
+- name: publish_q35sbsa_binaries
+  displayName: Publish QEMU Q35 SBSA Binaries
+  type: boolean
+  default: true
+
+name: $(Date:yyyyMMdd)$(Rev:.r) Mu Tiano Platforms NuGet Release
+
+jobs:
+  - job: Publish
+
+    strategy:
+      matrix:
+        ${{ if eq(parameters.publish_q35_binaries, true) }}:
+          QEMU Q35 Debug:
+            artifact_name: Binaries QemuQ35Pkg DEBUG
+            nuget_package_config_file: >-
+              https://raw.githubusercontent.com/microsoft/mu_tiano_platforms/main/Platforms/QemuQ35Pkg/NugetPublishing/DebugConfig.yaml
+            gcc_folder_name: Linux-x86

```

Comment:
I think the Linux-x86 might cause configuration given that this is building a X64 bios.

Should we consider changing it?

Reply:
Yes, I can change it. I'm also open to other organization, this was mainly done due to our existing patterns (which are based on external dependency rules not entirely relevant here).